presto-docs/src/main/sphinx/release/release-345.md

```diff
@@ -0,0 +1,47 @@
+# Release 345 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`concat_ws` function. ({issue}`4680`)
+* Add support for {func}`extract` for `time with time zone` values with precision other than 3. ({issue}`5539`)
+* Add dynamic filtering support for right joins. ({issue}`5555`) 
+* Add support for catalog session property to file based session property manager. ({issue}`5330`)
+* Add support for catalog session property to database based session property manager. ({issue}`5330`)
+* Improve query scalability by increasing broadcast join buffer size to 200MB.
+  Broadcast join buffer size can be set using `sink.max-broadcast-buffer-size` feature config. ({issue}`5551`)
+* Improve query performance by allowing larger dynamic filters. ({issue}`5529`)
+* Improve dynamic filtering across connectors by adding implicit cast support for dynamic filters. ({issue}`5461`)
+* Improve performance when encrypted spilling is enabled. ({issue}`5557`)
+ 
+## Security Changes
+
+* Add support for JSON Web Key (JWK) to the existing JSON Web Token (JWT) authenticator.  This is enabled by 
+  setting the `jwt.key-file` configuration property to a `http` or `https` url. ({issue}`5419`)
+* Add column security, column mask and row filter to file-based access controls. ({issue}`5460`)
+* Enforce access control for column references in `USING` clause. ({issue}`5620`)
+
+## Hive Connector Changes
+
+* Use `hive.domain-compaction-threshold` threshold for compacting dynamic filters. ({issue}`5365`)
+* Support reading Parquet timestamp encoded as microseconds. ({issue}`5483`)
+* Fix failure when reading Parquet `timestamp` columns encoded as `int64`. ({issue}`5443`)
+
+## MongoDB Connector Changes
+
+* Add support for adding a column. ({issue}`5512`)
```

Comment:
Maybe: `Add support for `ADD COLUMN`.  Not sure about this as the syntax is `ALTER TABLE name ADD COLUMN`

Reply:
I think it's fine to describe in English